crates/daphne-server/src/roles/leader.rs

```diff
@@ -165,16 +140,30 @@ impl crate::App {
                 .map_err(|e| fatal_error!(err = ?e, "failed to construct content-type header"))?,
         );
 
-        if let Some(bearer_token) = req.sender_auth.and_then(|auth| auth.bearer_token) {
+        if let Some(auth) = self
+            .service_config
+            .leader_to_helper_tokens
```

Comment:
`leader_to_helper_tokens`: Again, this is specific to taskprov, correct?

Reply:
No, this has nothing to do with taskprov, this is just for communication between the leader and the helper